x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/WaterfallContainer/Waterfall/SpanFlyout/index.tsx

```diff
@@ -153,6 +150,15 @@ export function SpanFlyout({
                 parentType="transaction"
               />,
               <>
+                {spanHttpStatusCode && (
+                  <HttpInfoContainer>
+                    <HttpInfoSummaryItem
+                      method={spanHttpMethod}
+                      url={spanHttpUrl}
+                      status={spanHttpStatusCode}
+                    />
```

Comment:
In this case if there is a url but not a status code neither will be displayed. I think the component should be rendered if there is either a status code or a url. 

You should make sure to check how it handles null values. I think it'll fails if `method` is `null`/`undefined`. 
Perhaps we should just use the same logic as in the transaction summary:
https://github.com/elastic/kibana/blob/a200a4ad7d977a00c7a2f18dc91919aa863a03e8/x-pack/legacy/plugins/apm/public/components/shared/Summary/TransactionSummary.tsx#L30-L35

I think it would be good if you can change this:
```js
    const method = idx(transaction, _ => _.http.request.method) || '';
```

to this:
```js
    const method = idx(transaction, _ => _.http.request.method);
```

and then allow `method` to be null by handling it inside `HttpInfoSummaryItem`